slash/react/src/Divider/Divider.tsx

```diff
@@ -1,20 +1,21 @@
 import classnames from "classnames";
 
 import "@axa-fr/design-system-slash-css/dist/Divider/Divider.css";
+import { ComponentProps } from "react";
 
-type DividerProps = {
+type DividerProps = ComponentProps<"hr"> & {
   mode?: "horizontal" | "vertical";
-  classModifier?: string;
 };
 
-export const Divider = ({
-  mode = "horizontal",
-  classModifier,
-}: DividerProps) => {
-  const componentClassName = classnames("af-divider", classModifier, {
-    "af-divider--horizontal": mode === "horizontal",
-    "af-divider--vertical": mode === "vertical",
-  });
+export const Divider = ({ mode = "horizontal", ...props }: DividerProps) => {
+  const componentClassName = classnames(
+    "af-divider",
+    {
+      "af-divider--horizontal": mode === "horizontal",
+      "af-divider--vertical": mode === "vertical",
+    },
+    props.className,
+  );
 
-  return <hr className={componentClassName} />;
+  return <hr className={componentClassName} {...props} />;
```

Comment:
attention className et toujours dans l'objet props et va écraser nos class,
```suggestion
export const Divider = ({ mode = "horizontal", className, ...props }: DividerProps) => {
  const componentClassName = classnames(
    "af-divider",
    {
      "af-divider--horizontal": mode === "horizontal",
      "af-divider--vertical": mode === "vertical",
    },
    className,
  );

  return <hr className={componentClassName} {...props} />;
```

Reply:
bien vu, merci @pplancq